submissions/MosQuitO404/js-dom/index.js

```diff
@@ -0,0 +1,76 @@
+import { teamsData } from "./data.js";
+
+const hamb = document.querySelector('#hamb');
+const popup = document.querySelector('#popup');
+const menu = document.querySelector('#menu');
+const popupMenu = document.querySelector('#menu').cloneNode(1);
+const placeholder = document.querySelector('#main-container');
+const body = document.body;
+
+// Hamburger menu
+
+hamb.addEventListener('click', hambHandler);
+    
+function hambHandler(e) {
+    e.preventDefault();
+    popup.classList.toggle('open');
```

Comment:
The same function calls as in `hambClose`

Reply:
It is necessary for responsive design, too. Fixed.